server/server.js

```diff
@@ -91,5 +116,5 @@ app.get('/server/status', (req, res) => {
 });
 
 app.get('*', function(req, res) {
-  res.sendFile(path.join(__dirname, '../client/build', 'index.html'));
+  res.sendFile(path.join(__dirname, '../client/build/index.html'));
```

Comment:
`res.sendFile(path.join(__dirname, '..', 'client', 'build', 'index.html'));`

Reply:
I think it looks better when we reduce the number of arguments, but that's a matter of personal opinion

Reply:
Fair enough.